app/controllers/orangelight/browsables_controller.rb

```diff
@@ -89,6 +94,7 @@ def populate_search_params(search_result)
       @search_term = search_term
       @exact_match = search_term == search_result.sort
       @match = search_result.id
+      @model == 'subjects' ? @match_vocabulary = search_result.vocabulary : nil
```

Comment:
Is this the same as?
```suggestion
      @match_vocabulary = search_result.vocabulary if @model == 'subjects'
```